components/locid/src/extensions/transform/fields.rs

```diff
@@ -107,18 +105,18 @@ impl Fields {
     ///     .expect("Failed to parse a Key.");
     /// let value: Value = "hybrid".parse()
     ///     .expect("Failed to parse a Value.");
-    /// let mut fields = Fields::from_vec_unchecked(vec![(key, value)]);
+    /// let mut fields: Fields = vec![(key, value)].into_iter().collect();
     ///
     /// let key: Key = "h0".parse()
     ///     .expect("Failed to parse a Key.");
     /// assert!(&fields.contains_key(&key));
     /// ```
-    pub fn contains_key<Q>(&self, key: Q) -> bool
+    pub fn contains_key<Q>(&self, key: &Q) -> bool
     where
-        Q: Borrow<Key>,
+        Key: Borrow<Q>,
+        Q: Ord,
```

Comment:
Here and below: I changed this function signature to match the signature of `LiteMap::contains_key`, which in turn appears to match the standard library `HashMap::contains_key`.